lib/item.rb

```diff
@@ -6,20 +6,31 @@ class Item
   attr_accessor :archived
 
   def initialize(objects, published_date, id = SecureRandom.uuid, archived: false)
-    @id = id
     @genre = objects[0]
     @author = objects[1]
     @source = objects[2]
     @label = objects[3]
     @published_date = published_date
+    @id = id
```

Comment:
I think we can improve the item class. Remember me to talk about this, I have some ideas...